.changeset/mighty-impalas-itch.md

```diff
@@ -0,0 +1,6 @@
+---
+'@cypherock/cysync-core-services': patch
+'@cypherock/cysync-desktop': patch
+---
+
+fix: evm token failing to migrate
```

Comment:
Remove changelog as this issue is not related to the customer release

Reply:
done